docs/en/ingest-management/release-notes/release-notes-8.0.asciidoc

```diff
@@ -12,15 +12,92 @@
 
 This section summarizes the changes in each release.
 
-//* <<release-notes-8.0.0>>
-
-coming[8.0.0]
+* <<release-notes-8.0.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.0.0 relnotes
+
+[[release-notes-8.0.0]]
+== {fleet} and {agent} 8.0.0
+
+Review important information about the {fleet} and {agent} 8.0.0 releases.
+
+[discrete]
+[[breaking-changes-8.0.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+[discrete]
+[[breaking-29458]]
+.{fleet-server} can no longer use username and password to authenticate with {es}
+[%collapsible]
+====
+*Details* +
+Starting in 8.0, {es} has removed write access to system indices by the
+superuser. Therefore, {fleet-server} can no longer use a username and password
+to authenticate with {es}. For more information, refer to
+{agent-pull}29458[#29458].
+
+*Impact* +
+If you run a self-managed {fleet-server} that authenticates with {es} through a
+username and password, you must update the configuration to use a {fleet-server}
+service token instead. For more information, see <<add-a-fleet-server>>.
+====
+
+[discrete]
+[[breaking-28006]]
+.`IncludeCreatorMetadata` removed from Kubernetes metadata config options
+[%collapsible]
+====
+*Details* +
+The deprecated and undocumented `IncludeCreatorMetadata` setting has been
+removed from the Kubernetes metadata config options. For more information, refer
+to {agent-pull}28006[#28006].
+
+*Impact* +
+This change is unlikely to impact existing users because the setting was never
+documented. However, if your config uses this setting, you must remove it now.
+====
+
+[discrete]
+[[new-features-8.0.0]]
+=== New features
+
+The 8.0.0 release adds the following new and notable features.
+
+//{fleet}::
+//* add info
```

Comment:
Let's add a note about Fleet supporting certificates generated by security on by default out-of-the-box. For the copy, maybe something like:
```
* Automatic configuration for trusting self-signed certificates generated by Elasticsearch
8.0+ for self-managed clusters. {kib-pull}120549[#120549]
```

It may make sense to link to the related Stack/ES doc: https://www.elastic.co/guide/en/elasticsearch/reference/8.0/configuring-stack-security.html. I also wonder if we should edit this ES doc to include a note that Fleet will configure Agents to automatically trust the CA fingerprint.

Reply:
Yeah let's update this sentence in that section with a note that Fleet will automatically be configured, original:

> Any clients that connect to Elasticsearch, such as the [Elasticsearch Clients](https://www.elastic.co/guide/en/elasticsearch/client/index.html), Beats, Logstash, and Fleet must validate that they trust the certificate that Elasticsearch uses for HTTPS.

Suggestion:

> Any clients that connect to Elasticsearch, such as the [Elasticsearch Clients](https://www.elastic.co/guide/en/elasticsearch/client/index.html), Beats, standalone Elastic Agents, and Logstash must validate that they trust the certificate that Elasticsearch uses for HTTPS. Fleet Server and Fleet-managed Elastic Agents will automatically be configured to trust the fingerprint of the CA.